data/virtualization/livemigration/host_15.xml.ep

```diff
@@ -0,0 +1,370 @@
+  </services-manager>
+  <software>
+    <install_recommended config:type="boolean">true</install_recommended>
+    <instsource/>
+    <packages config:type="list">
+      <package>dhcp-client</package>
+    % if ($check_var->('SYSTEM_ROLE', 'kvm')) {
+      <package>qemu-kvm</package>
+    % }
+    </packages>
+    <products config:type="list">
+      <product><%= uc $get_var->('SLE_PRODUCT') %></product>
+    </products>
+    <patterns config:type="list">
+      % for my $pattern (@$patterns) {
+      <pattern><%= $pattern %></pattern>
+      % }
+    </patterns>
+  </software>
+  <ssh_import>
+    <copy_config config:type="boolean">false</copy_config>
+    <import config:type="boolean">false</import>
+  </ssh_import>
+  <timezone>
+    <hwclock>UTC</hwclock>
+    <timezone>UTC</timezone>
+  </timezone>
+  <users config:type="list">
+    <user>
+      <authorized_keys config:type="list"/>
+      <encrypted config:type="boolean">true</encrypted>
+      <fullname>bernhard</fullname>
+      <gid>100</gid>
+      <home>/home/bernhard</home>
+      <home_btrfs_subvolume config:type="boolean">false</home_btrfs_subvolume>
+      <password_settings>
+        <expire/>
+        <flag/>
+        <inact/>
+        <max>99999</max>
+        <min>0</min>
+        <warn>7</warn>
+      </password_settings>
+      <shell>/bin/bash</shell>
+      <uid>1000</uid>
+      <user_password>$6$QSJG7nUenDgf$UcRX/Q6YWM2/S55RJ8Mgvu93oKEDlOs.7/rOTMYjjxXQPiZ/rq4ogx5EpkksVRZXoEF0kX8FeEa.gOTxl1l/t0</user_password>
+      <username>bernhard</username>
+    </user>
+    <user>
+      <authorized_keys config:type="list"/>
+      <encrypted config:type="boolean">true</encrypted>
+      <fullname>root</fullname>
+      <gid>0</gid>
+      <home>/root</home>
+      <home_btrfs_subvolume config:type="boolean">false</home_btrfs_subvolume>
+      <password_settings>
+        <expire/>
+        <flag/>
+        <inact/>
+        <max/>
+        <min/>
+        <warn/>
+      </password_settings>
+      <shell>/bin/bash</shell>
+      <uid>0</uid>
+      <user_password>$6$0bUrc6YvA/qw$h1Z3pzadaxmc/KgcHRSEcYoU1ShVNymoXBaRAQZJ4ozVhTbCvdAMbGQrQAAX7cC9cLRybhsvDio3kBX/IB3xj/</user_password>
+      <username>root</username>
+    </user>
+  </users>
+  <scripts>
+    <post-scripts config:type="list">
+      <script>
+        <filename>debug_libvirtd.sh</filename>
+        <source><![CDATA[
+cat >> /etc/libvirt/libvirtd.conf<< EOF
+log_level = 1
+log_filters="1:qemu 3:remote 4:event 3:util.json 3:rpc"
+log_outputs="1:file:/var/log/libvirt/libvirtd.log"
+EOF
+cp /usr/lib/systemd/network/99-default.link  /etc/systemd/network/99-default.link
+sed -i s/MACAddressPolicy=persistent/MACAddressPolicy=none/ /etc/systemd/network/99-default.link
+]]>
+        </source>
+      </script>
+    % if ($check_var->('VERSION', '15')) {
+      <script>
+        <filename>disable_dom0_balloon.sh</filename>
+        <source><![CDATA[
+sed -i '/autoballoon=/ s/.*/autoballoon="off"/' /etc/xen/xl.conf
```

Comment:
I read carefully the autoyst files fully again, very impressive(advanced autoyast usage, with limited code duplication among different guests and hosts) and considerate(handles many situations). It indeed an improvement to existing VT code about how autoyast is consumed!!!

Reply:
Thank you, Alice. 
Yes, the templates have some test data specific to this test scenario. Currently I know little about other VT test scenarios, did not take them into consideration while designing this migration. We can improve and extend them where needed in the future.